src/main/extension/extension_util.cpp

```diff
@@ -101,6 +101,11 @@ void ExtensionUtil::RegisterCollation(DatabaseInstance &db, CreateCollationInfo
 	auto data = CatalogTransaction::GetSystemTransaction(db);
 	info.on_conflict = OnCreateConflict::IGNORE_ON_CONFLICT;
 	system_catalog.CreateCollation(data, info);
+
+	// Also register as a function for serialisation
+	CreateScalarFunctionInfo finfo(info.function);
```

Comment:
Now that we are registering the collations in the catalog, maybe we should prefix them with `icu_collate_` or so? Otherwise we end up with a lot of 2-character function names which may well have conflicts with other functions (e.g. `de()`, `af()`, etc).